docs/reference/pf-command-reference.md

```diff
@@ -112,6 +112,18 @@ pf flow test --flow <path-to-flow-directory>
 
 Test the flow with single line from input file.
 
+```bash
+pf flow test --flow <path-to-flow-directory> --inputs inputs.jsonl
+```
+
+Test the flow from json file.
```

Comment:
put this the first ahead of the jsonl one

Reply:
fix it